Assets/Scripts/Game/UserInterface/UpDownSpinner.cs

```diff
@@ -174,7 +174,7 @@ public void SetRange(int minValue, int maxValue)
         public void SetValue(int value)
         {
             // Clamp values if ranges specified, otherwise accept whatever owner wants
-            if (minValue != 0 && maxValue != 0)
+            if (minValue != 0 || maxValue != 0)
```

Comment:
Why have you changed this? Are you sure that this doesn't change behaviour of any other UI window? This component is used in a few places.

Reply:
> Why have you changed this? Are you sure that this doesn't change behaviour of any other UI window? This component is used in a few places.

As the comment specifies, this code is meant to prevent the spinner from working if no range was set. If _either_ of the values have been set to non-zero, then it's been set. The way the code was written, a range like "0 to 60" would be considered "range not specified".

I did this so custom spellmaker windows (ex: my mod) could allow 0 as the minimum value, instead of 1.

I doubt this would break anything. This would only break a spinner that sets a non-zero maximum value yet somehow expects the spinner to be disabled.

From my look at the code, the only other place it's used is in StatsRollout.cs, and it sets no range, so both minValue and maxValue will be 0, which means it still works the same as before